measures/ResidentialAirflow/resources/unit_conversions.rb

```diff
@@ -132,7 +132,9 @@ def self.convert(x, from, to, fuel_type=nil)
       elsif from == 'btu/(lbm*r)' and to == 'j/(kg*k)' # by mass
         return x * 4187.0
       elsif from == 'btu/(ft^3*f)' and to == 'j/(m^3*k)' # by volume
-        return x * 67100.0 
+        return x * 67100.0
+      elsif from == 'btu/lbm*r' and to == 'wh/kg*k'
```

Comment:
Please use the same convention as other conversions:
```suggestion
      elsif from == 'btu/(lbm*r)' and to == 'wh/(kg*k)'
```